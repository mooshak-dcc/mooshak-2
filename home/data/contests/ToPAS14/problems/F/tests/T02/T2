108 5 9
AFFFAFFFF
AVVVVVXVA
AVVVIVXVA
AVVVVVTVF
AAAAAAAAA
8
E6
N2
W3
N6
S3
W2
N1
E8
