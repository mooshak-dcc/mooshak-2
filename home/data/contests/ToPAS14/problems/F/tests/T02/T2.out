Fora(10,2):137:D
